app/helpers/save_helper.rb

```diff
@@ -0,0 +1,7 @@
+module SaveHelper
```

Comment:
same for this as well. The name should be specific.

Reply:
Resolved it